csrc/dynamic_transform.cpp

```diff
@@ -454,8 +454,8 @@ void DynamicTransformConcretizer::concretize() {
   // Finally, propagate concretized domains
   auto all_stmts = StmtSort::getStmts(
       info_->fusion(),
-      /*traverse_members*/ false,
-      /*traverse_attributes*/ false,
+      /*traverse_members*/ true,
+      /*traverse_attributes*/ true,
```

Comment:
Since we propagate scalars, we should do this anywhere they might occur; namely in members and attributes. This was not necessary when we were only replacing `IterType`s since `IterDomain`s are traversed regardless of these settings.

Reply:
Didn't we discuss duplicated replacements if members were also traversed? Would it be a concern?